src/extensions/default/CodeFolding/main.less

```diff
@@ -63,11 +57,11 @@
     border: 1px solid @color-marker-border;
     color: @color-marker-text;
     background-color: @color-marker-bg;
-    
+
     &:after {
         content: "...";
         font-size: 1em;
         vertical-align: top;
         line-height: 0.8em;
     }
-}
```

Comment:
If you just need to roll back a single file: worry not! If you've accidentally deleted your files, you can get it back from any commit that had it previously. So if you're in the directory with main.css, and you do:
```
$> git checkout main.css master
```
you will get the `main.css` file back, as it existed in the master branch. Or, if you need it from commit hash 8236b721693df4h1263479f8d162:
```
$> git checkout main.css 8236b721693df4h1263479f8d162
```